package.json

```diff
@@ -11,5 +11,8 @@
   "devDependencies": {
     "cypress": "^13.3.0",
     "prettier": "3.0.3"
-  }
+  },
+  "cy:run:saucedemo": "cypress run --e2e --browser chrome --config-file cypress.saucedemo.config.js",
```

Comment:
Did you check these? I am not able to run them, i think they should go inside scipts block at line number 7

Reply:
Which command did you run? I successfully ran: `npx cypress run --browser chrome --config-file cypress.saucedemo.config.js` and `npx cypress run --browser chrome --config-file cypress.workco.config.js`

Reply:
I've just moved cy:run into scripts

Reply:
Yes but that is command it self and not an allais of the command. Idea of the allias for a script in the node package.json is to hide complex commands with multiple parameters behind one allias. In this case your allias should be cy:run:workco" I want to execute that instead of passig in whole command. 

See docs 
https://docs.npmjs.com/cli/v10/using-npm/scripts

Reply:
Alright, thanks! So, with moving cy:run alias into scripts and running: `npm run cy:run:saucedemo` or `npm run cy:run:workco` I am able to run tests for either site instead of passing multiple parameters. That's expected, right? 😄